bugout-dev/moonstream
# Add event provider and rewrite existing provider.

## Changes

Add event provider.
Add models wraper to db model.




## How to test these changes?

ensure canonical required
```
python -m moonstreamapi.admin.cli subtypes ensure-canonical
```

Usual request
```bash
curl --location --request GET 'http://127.0.0.1:7481/streams/?start_time=1639140929&end_time=1639141230&include_start=true&include_end=true' --header 'Authorization: Bearer $MOONSTREAM_ACCESS_TOKEN' | jq 
```








## Related issues
